hack/jenkins/job-configs/kubernetes-e2e.yaml

```diff
@@ -81,5 +83,69 @@
                 - cluster (k8s): ci/latest.txt<br>
                 - tests: ci/latest.txt
             timeout: 120
+        - 'gke-flaky':
+            test-owner: 'GKE on-call'
+            description: |
+                Run flaky e2e tests using the following config:<br>
+                - provider: GKE<br>
+                - apiary: staging<br>
+                - borg job: test<br>
+                - client (kubectl): ci/latest.txt<br>
+                - cluster (k8s): ci/latest.txt<br>
+                - tests: ci/latest.txt
+            timeout: 300
+    jobs:
+        - 'kubernetes-e2e-{suffix}'
+
+- project:
+    name: kubernetes-e2e-1.1
+    trigger-job: 'kubernetes-build-1.1'
+    test-owner: 'GKE on-call'
```

Comment:
Agreed, fixed. I also had some of  the descriptions wrong, that's fixed now too.